src/convertToObject.js

```diff
@@ -7,6 +7,35 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const mass1 = sourceString.replace(/\n/g, '').trim().split(';');
+  const mass2 = [];
+
+  for (let index = 0; index < mass1.length; index++) {
+    const element = mass1[index].trim();
+
+    mass2.push(element);
+  }
+
+  const strResalt = [];
+
+  for (let i = 0; i < mass2.length; i++) {
```

Comment:
```suggestion
  for (let i = 0; i < mass1.length; i++) {
```
but better to use for or, forEach 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach